gulpfile.js

```diff
@@ -78,3 +78,118 @@ const dev = series(build, serve, watcher);
 exports.build = build;
 exports.zip = series(build, zipper);
 exports.default = dev;
+
+// release imports
+const path = require('path');
+const releaseUtils = require('@tryghost/release-utils');
+
+let config;
+try {
+    config = require('./config');
+} catch (err) {
+    config = null;
+}
+
+const REPO = 'TryGhost/Casper';
+const USER_AGENT = 'Casper';
+const CHANGELOG_PATH = path.join(process.cwd(), '.', 'changelog.md');
+
+const changelog = ({previousVersion}) => {
+    const changelog = new releaseUtils.Changelog({
+        changelogPath: CHANGELOG_PATH,
+        folder: path.join(process.cwd(), '.')
+    });
+
+    changelog
+        .write({
+            githubRepoPath: `https://github.com/${REPO}`,
+            lastVersion: previousVersion
+        })
+        .sort()
+        .clean();
+};
+
+const previousRelease = () => {
+    return releaseUtils
+        .releases
+        .get({
+            userAgent: USER_AGENT,
+            uri: `https://api.github.com/repos/${REPO}/releases`
+        })
+        .then((response) => {
+            if (!response || !response.length) {
+                console.log('No releases found. Skipping');
+                return;
+            }
+
+            console.log(`Previous version ${response[0].name}`);
+            return response[0].name;
+        });
+};
+
+/**
+ *
+ * `yarn ship` will trigger `postship` task.
+ *
+ * [optional] For full automation
+ *
+ * `GHOST=2.10.1,2.10.0 yarn ship`
+ * First value: Ships with Ghost
+ * Second value: Compatible with Ghost/GScan
+ *
+ * You can manually run in case the task has thrown an error.
+ *
+ * `npm_package_version=0.5.0 gulp release`
+ */
+const release = () => {
+    // @NOTE: https://yarnpkg.com/lang/en/docs/cli/version/
+    const newVersion = process.env.npm_package_version;
+    let shipsWithGhost = '{version}';
+    let compatibleWithGhost = '2.10.0';
+    const ghostEnvValues = process.env.GHOST || null;
+
+    if (ghostEnvValues) {
+        shipsWithGhost = ghostEnvValues.split(',')[0];
+        compatibleWithGhost = ghostEnvValues.split(',')[1];
```

Comment:
Should we ensure against rogue `ghostEnvValues` here like
```
shipsWithGhost = ghostEnvValues.split(',')[0] || shipsWithGhost;
compatibleWithGhost = ghostEnvValues.split(',')[1] || compatibleWithGhost;
```
So if someone just writes `GHOST=2.x`, `compatibleWithGhost` is not `undefined`. Or maybe thats just too paranoid? 😄

Reply:
Thought about it, but didn't want to over optimise 🤣

Reply:
Ok pushed an improvement 👍

Reply:
😄 Yeah makes sense!